### DISCUSSION TITLE
Did nextjs polyfill fetch() in getStaticProps / getStaticPath ?

### DISCUSSION THREAD

**Discussion by iammarkps** [vercel/next.js#11995] [Help] [answered]
I want to remove axios from my site, before I will install any polyfill I want to know that did nextjs polyfill fetch() on server side too?

**Comment by ijjk** [comment, marked as answer]
Hi, Next.js does not currently polyfill `fetch` on the server, only on the client in browsers that don't have it natively as described in [our blog post](https://nextjs.org/blog/next-9-1-7#new-built-in-polyfills-fetch-url-and-objectassign) when this was added.

**Comment by Timer** [comment]
Do note, you can use server-only fetching packages in `getStaticProps` that are automatically eliminated from your client-bundle.
